docs/admin/upgrade.md

```diff
@@ -0,0 +1,27 @@
+# Upgrade guide
+
+## Why you might want to upgrade
+1. if a new version were released
+2. you have new features you have developed that you would like to move into production.
+
+Note: If you're adding your own development features, this guide assumes you have fully tested everything
+
+## Database Concerns
+It is _highly_ recommended to make a backup of your database before making any upgrades.  If the new version you are upgrading to has database schema changes, then you will need to upgrade your migrations file and remigrate your database.  This will most likely wipe your data.  It is recommended to start a new database alongside your current database and update your config file with the new database information.  Then you can copy your old data over.  __This should be tested before moving new code to production using a copy of your data__.  You have been warned.
+## How To Actually Upgrade
```

Comment:
You need a newline before the beginning of a header for it to render correctly in the docs:

```suggestion

 ## How To Actually Upgrade 
```